ricecooker/classes/nodes.py

```diff
@@ -427,6 +427,26 @@ def truncate_fields(self):
 
         super(TreeNode, self).truncate_fields()
 
+    def sort_topic_nodes(self, channel, key = None, reverse = False):
```

Comment:
Please name this `sort_child_nodes` and remove the `channel` argument. We should sort `self.children`.

Reply:
Also make sure to update the function doc strings to remove references to channel.

Reply:
changes have been made